src-plugins/manualRegistration/manualRegistrationToolBox.cpp

```diff
@@ -481,6 +503,11 @@ void manualRegistrationToolBox::disableSaveButtons(bool param)
     d->b_exportTransformation->setDisabled(param);
 }
 
+void manualRegistrationToolBox::disableComputeButton(bool param)
+{
+    d->b_computeRegistration->setDisabled(param);
+}
```

Comment:
I don't think so, it's homogeneity, and good coding for future dev with us (or just us)